src/test/shell/bazel/apple/bazel_apple_test.sh

```diff
@@ -40,101 +40,6 @@ function set_up() {
   create_new_workspace
 }
 
-function test_fat_binary_no_srcs() {
```

Comment:
Are these tests moving to apple_support with the toolchain, or are they no longer needed at all?

Reply:
these all moved to apple_support. anything that relied on the real toolchain and non macOS basic features has to move there. there are still many tests here using the mock toolchain for coverage